### PR TITLE
feat(Poke) : add presence

### PR DESCRIPTION
**Pull request by ashley0143**
## Description 
poke is a youtube front-end (like invidious) 
https://codeberg.org/ashley/poke/ 
https://github.com/ashley0143/poke/

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

File: websites/P/poke/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"description": {
+		"en": "Watch Silly videos or search the interwebs while being anonymous on poke!"
+	},
+	"url": [
+		"poketube.fun",
+		"poke.ggtyler.dev",
+		"poke.vern.cc"
+	],
+	"version": "1.0.0",
+	"logo": "https://cdn.glitch.global/d68d17bb-f2c0-4bc3-993f-50902734f652/yt-ukraine.png",
```
Comment (theusaf) [inline websites/P/poke/metadata.json:17]: [Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) must be 512x512px

File: websites/P/poke/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	"service": "Poke",
+	"description": {
+		"en": "Watch Silly videos or search the interwebs while being anonymous on poke!"
+	},
+	"url": [
+		"poketube.fun",
+		"poke.ggtyler.dev",
+		"poke.vern.cc"
+	],
+	"version": "1.0.0",
```
Comment (theusaf) [inline websites/P/poke/metadata.json:16]: There is a new field called `matches`, which is required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.

File: websites/P/poke/presence.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		);
+
+	return { startTimestamp, endTimestamp };
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://cdn.glitch.global/d68d17bb-f2c0-4bc3-993f-50902734f652/yt-ukraine.png?v=1703399674868",
+		smallImageKey: "more",
```
Comment (theusaf) [inline websites/P/poke/presence.ts:19]: Use image URLs rather than Discord asset names

File: websites/P/poke/presence.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+		case "privacy":
+			presenceData.details = "Managing preferences";
+			break;
+
+		case "watch": {
+			presenceData.smallImageKey =
+				document.querySelector("video") &&
+				!document.querySelector("video").paused
+					? "play"
+					: "pause";
```
Comment (theusaf) [inline websites/P/poke/presence.ts:43]:
```suggestion
					? Assets.Play
					: Assets.Pause;
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — CHANGES_REQUESTED** [review]
Ensure that the folder capitalization matches the service name. `Poke`, not `poke`

**Comment by ashley0143** [comment]
alr im gonna fix those

File: websites/P/Poke/presence.ts [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+			Math.floor(timeElement.duration)
+		);
+
+	return { startTimestamp, endTimestamp };
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://cdn.glitch.global/d68d17bb-f2c0-4bc3-993f-50902734f652/yt-ukraine.png?v=1703399674868",
```
Comment (theusaf) [inline websites/P/Poke/presence.ts:18]:
```suggestion
			"https://poketube.fun/css/yt-ukraine-512.png",
```

File: websites/P/Poke/presence.ts [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+		case "privacy":
+			presenceData.details = "Managing preferences";
+			break;
+
+		case "watch": {
+			presenceData.smallImageKey =
+				document.querySelector("video") &&
+				!document.querySelector("video").paused
+					? "play"
+					: "pause";
```
Comment (theusaf) [inline websites/P/Poke/presence.ts:42]:
```suggestion
					? Assets.Play
					: Assets.Pause;
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**